libr/include/r_cons.h

```diff
@@ -254,6 +254,10 @@ typedef struct r_cons_palette_t {
 	RColor graph_trufae;
 	RColor graph_traced;
 	RColor graph_current;
+        RColor graph_diff_match;
```

Comment:
btw wrong indent again.

Reply:
But it shows correct in <img width="1119" alt="Screen Shot 2019-08-16 at 10 54 09 AM" src="https://user-images.githubusercontent.com/20895544/63145257-576ab280-c014-11e9-82bc-ddb415ebca32.png">
and my ide :|